app/client/src/ce/utils/AnalyticsUtil.tsx

```
@@ -50,6 +54,30 @@ function logEvent(
   eventData?: EventProperties,
   eventType?: AnalyticsEventType,
 ) {
+  // Block tracking for anonymous users when the feature flag is enabled.
+  // We resolve store & selectors lazily to avoid circular dependencies.
+  try {
+    // Use eval("require") so webpack doesn't add this to the static dep graph
+    // preventing circular-chunk detection in prod build.
+    const { default: appStore } = eval("require")("store");
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)